src/resolvers/snapshot.ts

```diff
@@ -4,17 +4,20 @@ import { max } from '../constants.json';
 import { fetchHttpImage, axiosDefaultParams } from './utils';
 
 const HUB_URL = process.env.HUB_URL ?? 'https://hub.snapshot.org';
+const HUB_API_KEY = process.env.HUB_API_KEY ?? undefined;
 
 function createPropertyResolver(property: 'avatar' | 'cover') {
   return async (address: string) => {
     try {
+      const headers: Record<string, string> = HUB_API_KEY ? { 'x-api-key': HUB_API_KEY } : {};
```

Comment:
Do we really need to filter that ? Sending 'undefined' or empty string will be the same as sending nothing, afaik.

See https://github.com/snapshot-labs/snapshot-hub/blob/8e7e0a57689482ef52868f4fb875f3718ec183d0/src/helpers/keycard.ts#L11

Reply:
I think it is better not to send any extra headers, even if hub is already handling it, hub changes should not effect this code, we can still pass `undefined` here but `axios` doesn't accept 

```
No overload matches this call.
  Overload 1 of 2, '(config: AxiosRequestConfig<any>): AxiosPromise<any>', gave the following error.
    Type 'undefined' is not assignable to type 'string | number | boolean'.
  Overload 2 of 2, '(url: string, config?: AxiosRequestConfig<any> |
  ```

Reply:
I see.

One another way would be to add a filter on the `headers` inside the axios() call to filter out undefined, and forget about any filtering on the other part of the code.

But this is just extra, it's already working now

Reply:
> I see.
> 
> One another way would be to add a filter on the `headers` inside the axios() call to filter out undefined, and forget about any filtering on the other part of the code.
> 
> But this is just extra, it's already working now

I don't understand what you mean here 🤔 Could you provide an example?